syncthing/datadog_checks/syncthing/__about__.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.0'
+__version__ = '1.1.0'
```

Comment:
Could you also update the changelog accordingly?

Reply:
Done. Thanks!